src/vmm/src/builder.rs

```diff
@@ -448,6 +450,7 @@ pub fn build_microvm_from_snapshot(
     })?;
 
     // Build Vmm.
+    debug!("event_start: build microvm from snapshot");
```

Comment:
wouldn't we like a metric for this too?

Reply:
Hmm good point. But I am not sure if we have a good test candidate to generate those from. Should I remove those since they are unused.

Reply:
I think we can add the metrics as a follow up PR but this change would be good to have right now.